microsite/pages/en/community.js

```diff
@@ -0,0 +1,164 @@
+          <Block.Title>Community Sessions</Block.Title>
+          <Block.TextBox>
+            <Block.Paragraph>
+              <b>Adopters Community Sessions</b>
+              <br />
+              Backstage Community Sessions is the monthly meetup where we all
+              come together to listen to the latest maintainer updates, learn
+              from eachother about adopting, share exciting new demo's or
+              discuss any relevat topic like developer effectiveness, developer
+              experience, developer portals, etc.
+            </Block.Paragraph>
+
+            <Block.Paragraph>
+              <b>Contributor Community Sessions</b>
+              <br />
+              Discuss all things contributing, diving deep under the hood of
+              Backstage (Backstage of Backstage? Backerstage?). An open
+              discussion with maintainers and contributors of Backstage.
+            </Block.Paragraph>
+            <Block.LinkButton href="/on-demand">Watch now</Block.LinkButton>
+          </Block.TextBox>
+          <Block.TextBox>
+            <Block.Graphic
+              x={60}
+              y={12}
+              width={30}
+              src={`${baseUrl}img/b-sessions.png`}
+            />
+          </Block.TextBox>
+        </Block.Container>
+      </Block>
+
+      <Block small className="stripe bg-black-grey">
+        <Block.Container style={{ justifyContent: 'flex-start' }}>
+          <Block.TextBox>
+            <Block.Title>Backstage official Newsletter</Block.Title>
+            <Block.Paragraph>
+              The official monthly Backstage newsletter. Containing the latest
+              news from your favorite project.
+            </Block.Paragraph>
+            <Block.LinkButton href="https://mailchi.mp/spotify/backstage-community">
+              Subscribe
+            </Block.LinkButton>
+          </Block.TextBox>
+          <Block.Graphic
+            x={45}
+            y={12}
+            width={30}
+            src={`${baseUrl}img/news-fpo.png`}
+          />
+        </Block.Container>
+      </Block>
+
+      <Block className="stripe bg-black">
+        <Block.Container style={{ justifyContent: 'flex-start' }}>
+          <Block.TextBox>
+            <Block.Title>Spotlight</Block.Title>
+            <Block.Paragraph>
+              A recognition for valuable community work, the{' '}
+              <b>Contributor Spotlight</b>. Nominate contributing members for
+              their efforts! We'll put them in the spotlight ❤️.
+              <br />
+            </Block.Paragraph>
+            <Block.LinkButton href="nominate ">Nominate now</Block.LinkButton>
+          </Block.TextBox>
+          <Block.TextBox>
+            <Block.Title>Open Mic Meetup</Block.Title>
+            <Block.Paragraph>
+              A monthly casual get together of Backstage users sharing their
+              experiences and helping each other. Hosted by Roadie.io and
+              Frontside Software.
```

Comment:
might be nice to link to their respective home pages here?

Reply:
🤦‍♀️ yes!